src/btree/bt_delete.c

```diff
@@ -430,7 +447,8 @@ __instantiate_tombstone(WT_SESSION_IMPL *session, WT_PAGE_DELETED *page_del,
     /*
      * If we find an existing stop time point we don't need to append a tombstone. Such rows would
      * not have been visible to the original truncate operation and were, logically, skipped over
-     * rather than re-deleted.
+     * rather than re-deleted. (A row that _was_ visible would have forced its page to be slow-
+     * truncated rather than fast-truncated.)
```

Comment:
A row that _was_ visible would have forced its page to be slow-truncated rather than fast-truncated.

I don't understand this. Does it mean we should slow truncate a page if any row is visible to the truncate operation?

Reply:
I'm pretty sure what it was supposed to mean is: a row _with a stop time_ where the value was visible to the thread doing the truncate (that is, the stop time wasn't visible) would have forced a slow-truncate. Let me try to rewrite it so it's clearer. See [5431108](https://github.com/wiredtiger/wiredtiger/pull/8188/commits/5431108959e98cdf9b3aa1b0fb0aa532b7ab0db8)